sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -38,27 +36,34 @@ class SiteSQL(Base, CreatedMixin):
 
     site_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
     client_uuid = sa.Column(
-        UUID(as_uuid=True), sa.ForeignKey("clients.client_uuid"), nullable=False, default=uuid.uuid4
+        UUID(as_uuid=True),
+        sa.ForeignKey("clients.client_uuid"),
+        nullable=False,
+        default=uuid.uuid4,
     )
-    client_site_id = sa.Column(INTEGER)
+    client_site_id = sa.Column(sa.Integer)
     client_site_name = sa.Column(sa.String(255))
+
     region = sa.Column(sa.String(255))
     dno = sa.Column(sa.String(255))
     gsp = sa.Column(sa.String(255))
-    orientation = sa.Column(REAL)
-    tilt = sa.Column(REAL)
-    latitude = sa.Column(FLOAT, nullable=False)
-    longitude = sa.Column(FLOAT, nullable=False)
-    capacity_kw = sa.Column(REAL, nullable=False)
-    updated_utc = sa.Column(TIMESTAMP, nullable=False, default=lambda: datetime.utcnow())
+
+    # For metadata `NULL` means "we don't know".
```

Comment:
Having possible `NULL` values for metadata seemed like a requirement.

Reply:
yea, lets get this through, and we can always come back to it. There was a request for `capacity_kw` o be null